fastlane/Fastfile

```diff
@@ -212,21 +222,56 @@ end
 after_all do |lane|
   next unless is_ci?
 
+if lane == :beta then
```

Comment:
You could also give another `fields` array for the beta lane.

For example:
```ruby
fields = [{
    title: "CHANGELOG",
    value: "#{changelog}",
    short: false
}]

# Add the HockeyApp information when in beta lane.
fields << {
    title: "Hockey Link",
    value: "#{hockeyLink}",
    short: true
} if lane == :beta

...
attachment_properties: {
    title: "#{_OLLIE_NAME_} beta",
    fields: fields
}
...
```

This way you remove the duplicate code.

Much 💕 for Ruby!